### PR TITLE
updates webpack

### LINKED ISSUE #86: Update Webpack Version
Currently webpack version 1.15. 

I am having issues with resolving #85 (adding storybook to frost-web) and I believe it is due to the outdated webpack version.

### PR DESCRIPTION
**Pull request by warrenv**
This commit updates webpack to the latest 3.x version.

Fixes #86

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -93,55 +95,60 @@ module.exports = {
...
-      { test: /\.tsx?$/, loaders: production
-        ? ['babel', 'awesome-typescript-loader']
-        : ['react-hot', 'babel', 'awesome-typescript-loader'] },
+    rules: [
+      { test: /\.tsx?$/, use: production
+        ? ['babel-loader', 'awesome-typescript-loader']
+        : ['react-hot-loader', 'babel-loader', 'awesome-typescript-loader'] },
+      // {
+      //   test: /\.s?css$/, use: [ 'style-loader', 'css-loader?sourceMap&importLoaders=1', 'postcss-loader', 'sass-loader?sourceMap' ]
+      // },
```
Comment (jwicks31) [inline webpack.config.js]: Not sure if you meant to leave this comment in here.

File: webpack.config.js [hunk 2]
```diff
@@ -11,22 +11,19 @@ const validEnvironments = [
...
 ]
-const environment = process.env.NODE_ENV
+const environment = process.env.NODE_ENV || 'development'
 
 assert(validEnvironments.includes(environment), `Invalid value for NODE_ENV: ${environment}. Valid values are: ${validEnvironments}`)
 
 const production = environment === 'production'
 const development = environment === 'development'
 const configurationPath = `./env/${environment}.json`
-const redirects = `./_redirects.${environment}`
```
Comment (jwicks31) [inline webpack.config.js]: I think to fix the redirect issue for Netlify this line(and probably should include the console.log below) need to be added back in.

**Comment by wzalazar** [comment]
@warrenv 

1 - **_redirect** file was fixed
2 - Hot reloading still not working, but maybe we can track this issue in the backlog
3 - I saw you set **html-webpack-plugin** in **2.30.1**, maybe we have to put **html-webpack-plugin** in **greenkeeper ignore** for avoiding updates

**Comment by wzalazar** [comment]
and what do you say about **html-webpack-plugin** ?

**Comment by warrenv** [comment]
I moved it into greenkeeper.ignore

**Pull request merged**